src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -668,7 +668,7 @@ void SurgeGUIEditor::idle()
          }
       }
 
-      for (int i = 0; i < 8; i++)
+      for (int i = 0; i < n_fx_slots; i++)
```

Comment:
This 8 is not n_fx_slots. It is the size of the ctrl_refresh_queue which doesn't have a constant.